Source/Scene/parseKTX2.js

```diff
@@ -0,0 +1,335 @@
+import defined from "../Core/defined.js";
```

Comment:
I'll hold off on reviewing this until you evaluate [KTX-parse](https://github.com/donmccurdy/KTX-Parse)